From 61a79dda62aa64edd1648ca97d4fe01ce9f69811 Mon Sep 17 00:00:00 2001
From: Lars Kanis <lars@greiz-reinsdorf.de>
Date: Thu, 10 Jun 2021 21:10:33 +0200
Subject: [PATCH] Don't add an exe file manifest

It is later added as part of the rubyinstaller2 build process.
The manifest was introduced in: https://github.com/ruby/ruby/pull/4505
---
 win32/resource.rb | 1 -
 1 file changed, 1 deletion(-)

diff --git a/win32/resource.rb b/win32/resource.rb
index bc3eda5630..e0e7323dc7 100755
--- a/win32/resource.rb
+++ b/win32/resource.rb
@@ -61,7 +61,6 @@ def icons.find(path)
 #include <winver.h>
 
 #{icon || ''}
-#{type == 'VFT_APP' ? "1 RT_MANIFEST ruby.manifest" : ""}
 VS_VERSION_INFO VERSIONINFO
  FILEVERSION    #{nversion}
  PRODUCTVERSION #{nversion}
-- 
2.30.0.windows.1

